components/ui/vertical-navigation.tsx

```
@@ -0,0 +1,309 @@
+                          `.desktop-nav-button:nth-of-type(${
+                            index + 2
+                          })`
+                        ) as HTMLElement;
+                        nextButton?.focus();
+                      }
+                      break;
+                    case 'Home':
+                      e.preventDefault();
+                      const firstButton = document.querySelector(
+                        `.desktop-nav-button:nth-of-type(1)`
+                      ) as HTMLElement;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)